src/Raven.Client/Documents/Linq/RavenQueryProviderProcessor.cs

```diff
@@ -2896,6 +2908,13 @@ private void HandleSelectTimeSeries(MethodCallExpression callExpression, string
             AddToFieldsToFetch(path, alias);
         }
 
+        private void HandleSelectId( string alias = null)
+        {
+            string path = $"{Constants.Documents.Indexing.Fields.DocumentIdFieldName}";
```

Comment:
assuming that this can be without `$""` just: `string path = Constants.Documents.Indexing.Fields.DocumentIdFieldName` ?

Reply:
Done.